### ISSUE TITLE
[1.16.4] Undergarden Tools are not Tetra compatible

### ISSUE THREAD

**Issue by NielsPilgaard** [Syrikal/Tetra-Undergarden-Patch#1]
The various tools from Tetra should be added as replacements, in order to allow the mod's tools to be modified with Tetra

example:
```json
{
        "predicate": { "item": "mekanismtools:bronze_sword" },
        "item": "tetra:modular_sword",
        "modules": {
            "sword/blade": ["sword/basic_blade", "basic_blade/bronze"],
            "sword/hilt": ["sword/basic_hilt", "basic_hilt/stick"],
            "sword/pommel": ["sword/decorative_pommel", "decorative_pommel/bronze"],
            "sword/guard": ["sword/makeshift_guard", "makeshift_guard/bronze"]
        },
        "improvements": {
            "sword/blade:workable": 1,
            "sword/pommel:workable": 1,
            "sword/guard:workable": 1
        }
    }
```

**Comment by Syrikal** [comment]
Fixed in 1.1.2.

**Comment by NielsPilgaard** [comment]
Thanks!
